### ISSUE TITLE
Toolbar.setBackCommand() on Javascript

### ISSUE THREAD

**Issue by jsfan3** [codenameone/CodenameOne#2634]
I see that now the Javascript port loads an Android native theme or an iPhone native theme according to the used device, that's great.

But now there is a side effect: a consequence of this new feature is that `Toolbar.setBackCommand` is not usable on Android (with the following code), because the hardware back button is not mapped to the back command. On iPhone I can invoke the BackCommand on the left of the Title, but on Android I expect to invoke it on the hardware button.

```
this.getToolbar().setBackCommand(backCmdTitle, Toolbar.BackCommandPolicy.ONLY_WHEN_USES_TITLE, l -> {
                previousForm.showBack();
});
```

**Comment by jsfan3** [comment]
Can you map the back command to the hardware back button, on the Javascript port?

**Comment by shannah** [comment]
Good point.  I should be able to tap into the browser navigation for the back command - and I believe the hardware back button on Android maps to the browser navigation (i.e. browser back).

**Comment by jsfan3** [comment]
On the Friday of the next week or today? I'm asking because, with a new Javascript build, the browser back button is not usable with `Toolbar.BackCommandPolicy.ONLY_WHEN_USES_TITLE`. Tested with Chrome on Android.

**Comment by codenameone** [comment]
It should be deployed @shannah can you verify that your change went in?
